APMrover2/radio.cpp

```diff
@@ -77,10 +77,8 @@ void Rover::rudder_arm_disarm_check()
             // not at full right rudder
             rudder_arm_timer = 0;
         }
-    } else if (!motor_active() && !g2.motors.have_skid_steering()) {
+    } else if (!motor_active()) {
```

Comment:
motor_active is true if we get steering, so we will never reach channel_steer->get_control_in()  in manual and steering mode no ?

Reply:
@khancyr, thanks for reviewing.
Yes, that's right.  So it's generally not possible to disarm in steering and manual modes because as the pilot tries to hold the stick left, a steering command will be sent to the motors and it will become "motor_active".